pipelines/lib/util.groovy

```diff
@@ -385,7 +385,7 @@ def lsstswBuild(
       }
   } // runEnv
 
-  def agent = null
+  def agent = lsstswConfig.label
```

Comment:
Isn't this unnecessary, given that it's set in line 394?  Or you could remove 394.